### DISCUSSION TITLE
Welcome to CAN_Commander Discussions!

### DISCUSSION THREAD

**Discussion by MatthewKuKanich** [MatthewKuKanich/CAN_Commander#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kjv1611isch** [comment]
Thank you Matt for all your hard work

**Comment by EdaDyckATpmDOTme** [comment]
this is great. when and where to buyÉ

**Comment by Richatk01** [comment]
So I had a couple questions and your intelligence might be able to help. I don't know who to look for where to look or how to go about getting started.

I work on tractors and they have a engine control unit with can bus high and low I want to grab some information from the ECU via can and loop it back around and tell it it has just that so it desires a certain fan speed at all times while the engine is running and I wanted to grab that information somehow and turn around and tell it it has just that would you be able to help? Where can I look? How can I start developing a product to get this going? I would appreciate any input thank you.

**Comment by fcercwer3** [comment]
When and where can one buy a CAN commander for the flipper zero?

**Reply by liltigerlilie** [reply]
Early bird deposit special just dropped on Rabbit Labs!! Good until 1/2/25.

**Comment by lovecomission** [comment]
Hi Matthew. Just came across a video of yours on youtube about your CAN commander. Have not finished watching or researching everything. Before I go down your rabbit hole, can you give me a quick overview of what your CAN commander is? Do I buy an Arduino board first and then load your code onto it? Can I install an OBD2 port plug onto the device you recommend? I am looking at another device that works as a CAN sniffer as well. I am going to be reverse engineering the CAN channels in different cars so that I can pull certain driver input and vehicle data to analyze in telemetry software as a driver coach.
